src/test/java/uk/gov/hmcts/reform/cpo/service/CasePaymentOrdersServiceImplTest.java

```diff
@@ -1,32 +1,118 @@
 package uk.gov.hmcts.reform.cpo.service;
 
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.cpo.exception.CasePaymentOrderCouldNotBeFoundException;
 import uk.gov.hmcts.reform.cpo.repository.CasePaymentOrdersRepository;
 import uk.gov.hmcts.reform.cpo.service.impl.CasePaymentOrdersServiceImpl;
 
-import static org.springframework.test.util.AssertionErrors.assertNotNull;
+import java.util.List;
+import java.util.UUID;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.cpo.service.impl.CasePaymentOrdersServiceImpl.AUDIT_ENTRY_DELETION_ERROR;
+
+@SuppressWarnings("PMD.JUnitAssertionsShouldIncludeMessage")
```

Comment:
PMD has been removed so this check is no longer enforced. So feel free to remove.